src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -258,6 +317,17 @@
         }
     }
 
+    .datagrid-action-bar {
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: stretch;
+        width: 100%;
```

Comment:
Yes I'm thinking we can revisit this because we are recommending the use of button groups in the action bar, and the button groups themselves are supposed to be responsive as far as I know. But the button groups haven't been implemented yet. Of course the action bars aren't constrained to contain just button groups even if we recommend it, so we should definitely raise this for more UX input. cc @yenmade @reddolan